ast/ast.go

```diff
@@ -166,7 +166,7 @@ func (a *ActionExpr) Pos() Pos { return a.p }
 
 // String returns the textual representation of a node.
 func (a *ActionExpr) String() string {
-	return fmt.Sprintf("%s: %T{Expr: %v, Code: %v}", a.p, a, a.Expr, a.Code)
+	return fmt.Sprintf("%s: %T{Expr: %v, Code: %v, FuncIx: %d, ID: %p}", a.p, a, a.Expr, a.Code, a.FuncIx, a)
```

Comment:
I had added this for easier debugging (and noticing the missing clone), let me know if you'd rather have it removed (this could be considered a breaking change, but I think that's a stretch :) )

Reply:
Please remove it.

Reply:
Done.